1.8/administration/overlay-networks/isolation.md

```diff
@@ -0,0 +1,73 @@
+---
+layout: page
+nav_title: Isolation
+post_title: Configuring Isolation in Overlay Networks
+menu_order: 20
+---
+
+# iptables rules
+
+DC/OS uses [iptables](http://linux.die.net/man/8/iptables) to set up overlay network isolation. iptables are a high-speed, built-in mechanism for filtering traffic in Linux systems. We recommend configuring filtering by deploying a homogenous set of rules to all nodes in your infrastructure. In order to simplify this, we also recommend using the [ipset](http://ipset.netfilter.org/ipset.man.html) feature of iptables. 
+
+Set up your own chain that jumps from the `FORWARD` chain. You can do this by running the following command:
+
+    $ iptables -N dcos-isolation
+
+We want to set up a default deny or a default accept policy between filtered overlays. To set up default deny, run the following:
+
+    $ iptables -A dcos-isolation -j REJECT
+
+To set up default accept, run the following:
+
+    $ iptables -A dcos-isolation -j RETURN
+
+We recommend using the `REJECT` directive as opposed to the `DROP` directive as it makes troubleshooting easier. The default is to allow all.
+
+Use ipset to get onto the isolation chain. Create a `hash:net` type ipset named `overlays` that has all of the overlay networks that you want to restrict traffic from, or to. Then insert the rule:
+
+    $ iptables -I FORWARD -m set --match-set overlays src -m set --match-set overlays dst -j dcos-isolation
+
+This rule says that if a given packet is from any of the overlays and is destined to any other overlay, send it to the `dcos-isolation` rule. In most environments, the system should prevent an overlay network's outbound packets from re-entering the same overlay network. In order to avoid this, add an exception set of type `hash:net,net` and add entries for networks that should not be filtered. Modify the rule to: 
```

Comment:
How about "in most environments, the system does not prevent an overlay network's outbound packets from reentering the same overlay network. In order to prevent this..."?

And why would I want to ensure this doesn't happen?